python-modules/cis_profile/cis_profile/profile.schema

```diff
@@ -1,109 +1,173 @@
                 "ssh_public_keys": {
-                    "$ref": "#/definitions/StandardAttributeValues"
+                    "$ref": "#/definitions/StandardAttributeValues",
+                    "$comment": "List of known public OpenSSH keys for the user."
                 },
                 "pgp_public_keys": {
-                    "$ref": "#/definitions/StandardAttributeValues"
+                    "$ref": "#/definitions/StandardAttributeValues",
+                    "$comment": "List of known PGP/GPG public keys for the user."
                 },
                 "access_information": {
-                    "$ref": "#/definitions/AccessInformation"
+                    "$ref": "#/definitions/AccessInformationValuesArray",
+                    "$comment": "List of publisher-specific information that is used to authorize or deny access for this user"
                 },
                 "fun_title": {
-                    "$ref": "#/definitions/StandardAttributeString"
+                    "$ref": "#/definitions/StandardAttributeString",
+                    "$comment": "A title the user likes to use, if any. Not necessarily a real job title."
                 },
                 "description": {
-                    "$ref": "#/definitions/StandardAttributeString"
-                },
-                "location_preference": {
-                    "$ref": "#/definitions/StandardAttributeString"
+                    "$ref": "#/definitions/StandardAttributeString",
+                    "$comment": "A description field for the user. Could be used for a biography, or anything the user would like to say."
                 },
-                "office_location": {
-                    "$ref": "#/definitions/StandardAttributeString"
+                "location": {
+                    "$ref": "#/definitions/StandardAttributeString",
+                    "$comment": "The location the user declares to be living at (city, country, etc.), if any."
                 },
                 "timezone": {
-                    "$ref": "#/definitions/StandardAttributeString"
+                    "$ref": "#/definitions/StandardAttributeString",
+                    "$comment": "The timezone the user declares to be living at, if any."
                 },
-                "preferred_languages": {
-                    "$ref": "#/definitions/StandardAttributeValues"
+                "languages": {
+                    "$ref": "#/definitions/StandardAttributeValues",
+                    "$comment": "The languages the user would like to use to communicate with others."
                 },
                 "tags": {
-                    "$ref": "#/definitions/StandardAttributeValues"
-                },
-                "pronouns": {
-                    "$ref": "#/definitions/StandardAttributeString"
+                    "$ref": "#/definitions/StandardAttributeValues",
+                    "$comment": "Just tags. Used for Mozillians.org style vouches, for example."
```

Comment:
This can be a bit misleading. Regardless of vouches, which may be an access group, I would describe this something like a union of interests. Eg python tag is a self descriptive tag for people with a common interest in python